### DISCUSSION TITLE
TCP or UDP  Is there any sense that I miss?

### DISCUSSION THREAD

**Discussion by Taxom** [jvde-github/AIS-catcher#104] [Q&A] [answered]
Sorry for the stupid question, I wonder if it makes sense to use TCP instead of UDP when sending data to Marinetraffic? What is this given except for the additional load of the processor and increase the volume of traffic?
That is, is it worth using TCP if the UDP works fine?

Thanks.

**Comment by jvde-github** [comment, marked as answer]
Hi, the short answer is that we didn't find any benefits. MT closes all TCP connections every 3 hours, after that the program reconnects and continues. So, you have slightly more comfort that the messages are received and the computers are talking. But you can see that in the summary page of MT as well. Did not see advantages in reliability in our short testing done.
